css/style.css

```diff
@@ -94,3 +95,325 @@ header a {
+  width: 85%;
+}
+
+.viewpro-btn {
+  padding: 12px;
+  text-decoration: none;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  width: 124px;
+  height: 48px;
+  color: #396df2;
+  text-align: center;
+}
+
+.viewpro-btn:hover {
+  background-color: #396df2;
+  color: #fff;
+}
+
+.resume-button {
+  padding: 12px;
+  text-decoration: none;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  width: 124px;
+  height: 48px;
+  color: #396df2;
+  text-align: center;
+}
+
+.resume-button:hover {
+  background-color: #396df2;
+  color: #fff;
+}
+
+.card .card-sub-head {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 260px;
+  font-weight: 600;
+  color: #344563;
+  padding: 0 2px;
+}
+
+/* About section */
+.aboutme {
+  background-color: #fff;
+  padding-bottom: 114px;
+  display: flex;
+  flex-direction: column;
+  border-top-right-radius: 80px;
+}
+
+.aboutme .header {
+  background-image: url("mobile-images/works/aboutmyself.png");
+  border-top-right-radius: 80px;
+}
+
+.aboutme .header .lets-connect {
+  text-decoration: none;
+  color: #7f8cff;
+  padding-top: 3px;
+  padding-bottom: 23px;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.aboutme .header .about-title {
+  font-family: "Poppins", sans-serif;
+  font-weight: bold;
+  font-size: 32px;
+  color: #172b4d;
+  padding: 10px 0;
+  margin: 0;
+  padding-right: 45%;
+  line-height: 44px;
+}
+
+.aboutme .header ul {
+  margin-left: 15px;
+}
+
+.aboutme .header .resume-button {
+  padding: 12px;
+  width: 162px;
+  height: 48px;
+  margin-top: 20px;
+}
+
+.aboutme .header .resume-button:hover {
+  color: #fff;
+  background-color: #7f8cff;
+}
+
+.aboutme .languages {
+  display: grid;
+}
+
+.aboutme .languages .lang-item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 35px;
+  height: 72px;
+}
+
+.aboutme .languages .lang-item h3 {
+  font-family: "Poppins", sans-serif;
+  font-size: 26px;
+  font-weight: 500;
+  line-height: 23px;
+}
+
+.aboutme .languages .lang-item h4 {
+  font-family: "Poppins", sans-serif;
+  font-size: 25px;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.aboutme .languages .lang-item .down-arrow {
+  width: 14px;
+  height: 8px;
+  color: #505f79;
+}
+
+.aboutme .languages .lang-item .right-arrow {
+  width: 8px;
+  height: 14px;
+  color: #505f79;
+}
+
+.aboutme .languages hr {
+  color: #dfe1e6;
+  height: 1px;
+  margin: 2px 30px;
+  text-align: center;
+}
+
+.aboutme .languages ul {
+  margin: 0;
+  padding: 0;
+}
+
+.aboutme .languages .content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 12px;
+  margin: 0 24px;
+}
+
+.aboutme .languages .content div {
+  display: flex;
+  align-items: center;
+  height: 66.72px;
+  background-color: #f7f7f9;
+  border-radius: 8px;
+  padding: 0 14px;
+  font-family: "Poppins", sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  color: #253858;
+}
```

Comment:
kindly according to the requirement of the project, the font must be similar to the design of the Figma, please change the font of your text so that it gives good visibility of the text